.circleci/config.yml

```diff
@@ -298,6 +334,12 @@ workflows:
             parameters:
               python-version: *python-versions
 
+      - test-cli:
+          name: test-cli-linux-py<< matrix.python-version >>
+          matrix:
+            parameters:
+              python-version: &python-versions ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
Can't this use the same reference as above (`*python-versions`)?

Reply:
of course it can :smiley:  good catch, thanks!